dockerfiles/base/scripts/base/commands/cmd_stop.sh

```diff
@@ -0,0 +1,59 @@
+#!/bin/bash
+# Copyright (c) 2012-2016 Codenvy, S.A.
+# All rights reserved. This program and the accompanying materials
+# are made available under the terms of the Eclipse Public License v1.0
+# which accompanies this distribution, and is available at
+# http://www.eclipse.org/legal/epl-v10.html
+#
+# Contributors:
+#   Tyler Jewell - Initial Implementation
+#
+
+cmd_stop() {
+  debug $FUNCNAME
+  FORCE_STOP=false
+  if [[ "$@" == *"--skip:graceful"* ]]; then
+  	FORCE_STOP=true
+  fi
+
+  if server_is_booted $(get_server_container_id $CHE_CONTAINER_NAME); then 
+    if [[ ${FORCE_STOP} = "false" ]]; then
+      info "stop" "Stopping workspaces..."
+      if ! $(cmd_lifecycle action "graceful-stop" "$@" >> "${LOGS}" 2>&1 || false); then
```

Comment:
The format can be consistent. It's just that it is changing for now a lot.
We could show raw output in case of error. BTW if there is an error user might need to see the output. So it's like a shortcut.

We can also return internal error code
0 = ok
409 : auth problem, etc so in cli we can say that there is an auth issue, etc